templates/resources/integration.md.tmpl

```diff
@@ -0,0 +1,17 @@
+---
+layout: "fastly"
+page_title: "Fastly: integration"
+sidebar_current: "docs-fastly-resource-integration"
+description: |-
+  Provides a Fastly integration. Fastly will use these integrations to send you notifications and alerts related to the Fastly product. Mailing list, Microsoft Teams, New Relic, PagerDuty, Slack, and webhook are supported as integrations.
+---
+
+# fastly_integration
+
+Provides a Fastly integration. Fastly will use these integrations to send you notifications and alerts related to the Fastly product. Mailing list, Microsoft Teams, New Relic, PagerDuty, Slack, and webhook are supported as integrations.
```

Comment:
```suggestion
Provides a Fastly integration. Fastly will use these integrations to send you notifications and alerts related to the Fastly product. Mailing list, Microsoft Teams, New Relic, PagerDuty, Slack, and webhook are supported as integrations.

~> **IMPORTANT:** The mailing list integrations require confirmation. To send a confirmation email and verify integration status, after applying changes using Terraform, please visit https://manage.fastly.com/observability/alerts/integrations
```

This will be rendered in a nice yellow box with appropriate emoji icon (example: https://registry.terraform.io/providers/fastly/fastly/latest/docs/resources/secretstore)